src/components/MoneyInput/index.js

```diff
@@ -69,12 +69,17 @@ export function MoneyInput({
     isCurrencyControlled || props.initialSelectedCurrency
       ? currencies.indexOf(props.currency || props.initialSelectedCurrency)
       : 0
+  const initialValue = props.initialAmount ? props.initialAmount / 100 : 0
```

Comment:
You can add the property `initialAmount` to the function prop arguments, so that way you don't have to repeat `props.initialAmount` every time, and just use `initialAmount`.